app/components/AnswerRow/AnswerRow.jsx

```diff
@@ -40,15 +39,29 @@ function AnswerRow({ searchTerm, isPreview, isQuestionModalOpen, ...props }) {
       )}
     </Styled.AnswerRow>
   );
+  
+  const renderAnswerLabel = () => {
+    let txt_answer = 'Best Answer';
+    if(props.isCommentApproved) {
+      txt_answer = `Comment approved by ${props.approver.full_name}`;
+    }
+    else if (props.isCommunityAnswer) {
+      txt_answer = 'Community answer';
+    };
+    
+    return (
+      <Label type={'Answer'} text={txt_answer} />
+    )
+  }
```

Comment:
With the previous mentioned changes can do something like this:
```suggestion
  const renderAnswerLabel = () => {
    if(props.isCommentApproved) {
      return <Label type={'Answer'} text={'Approved'} approvedBy={props.approver.full_name} />
    }
    else if (props.isCommunityAnswer) {
      return <Label type={'Answer'} text={'Community answer'} />
    };
    return (
      <Label type={'Answer'} text={'Best Answer'} />
    )
  }
```

Reply:
Would look like this:

<img width="666" alt="Captura de Pantalla 2022-11-16 a la(s) 14 31 50" src="https://user-images.githubusercontent.com/71342528/202288070-1c9eb907-299b-4462-989b-22a215ea310d.png">

Reply:
Done!